tools/install.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+source "getopts_long.bash"
```

Comment:
That won't work with `curl` though, right? Or am I wrong here? 🤔

Reply:
oh my got - you are probably right. So lets stick with short args?

Reply:
I'd say yes - just basic `-h`, `-d` would be more than enough IMO